File: webproject/src/main/java/com/mxy/bbs_server/entity/Post.java
package com.mxy.bbs_server.entity;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

import java.util.List;


@Data
@AllArgsConstructor
@Builder
public class Post {
    private String id;
    private String date;
    private String owner;
    private String title;
    private String content;
    private List<String> images;
    private Integer likeNum;
    private List<String> reviews;
    private String PostType;
}

File: AndroidClient/app/src/main/java/com/example/client/programLogic/bean/Post/Post.kt
package com.example.client.programLogic.bean.Post

class Post {
    lateinit var id: String
    lateinit var date: String
    var title: String
    var content: String
    lateinit var likeNum: Integer
    lateinit var PostType: String
    var images: ArrayList<String> ? =null
    var owner: String
    val reviews: List<String>? = null

    constructor(title: String, content: String, ownerUsername: String) {
        this.title = title
        this.content = content
        this.owner = ownerUsername
    }

}